ts/parts/Utilities.ts

```diff
@@ -1337,7 +1337,7 @@ function isNumber(n: unknown): n is number {
  *
  * @return {void}
  */
-H.erase = function (arr: Array<any>, item: any): void {
+function erase(arr: Array<any>, item: any): void {
```

Comment:
Maybe it is better to replace the `any` types with `unknown`, which should be similar for TypeScript.

Reply:
I agree, please see latest changes.